apps/ui/src/views/Proposal.vue

```diff
@@ -191,6 +200,42 @@ watchEffect(() => {
                 </UiButton>
               </UiTooltip>
             </div>
+            <div v-else-if="proposal.type === 'single-choice'" class="flex flex-col space-y-2 py-2">
```

Comment:
Maybe it's time to extract each voting type form into their individual component ?

Reply:
Yeah, I think it's good idea to extract those.